src/gengowatcher/ui_textual.py

```
@@ -1845,14 +2397,385 @@ def _dashboard_refresh_targets(self) -> list[tuple[type, str]]:
+            from .web import start_web_server_thread
+
+            thread = start_web_server_thread(
+                host=host,
+                port=port,
+                config=self.config,
+                state=self.state,
+                logger=self._log_source,
+                watcher=self.watcher,
+                start_watcher_thread=False,
+            )
+        except Exception as error:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)